buildpack.md

```diff
@@ -1105,17 +1101,28 @@ The `[[buildpack.licenses]]` table is optional and MAY contain a list of buildpa
 *Key: `sbom-formats = [ "<string>" ]`*
  - MUST be supported SBOM media types as described in [Software-Bill-of-Materials](#software-bill-of-materials).
 
-#### Stacks
+#### Targets
 
-A buildpack descriptor may specify `stacks`.
+A buildpack descriptor MAY specify `targets`.
```

Comment:
what if it doesn't? --  besides backwards compatibility, what does it mean for this to be optional?

Reply:
Good point... maybe this should be SHOULD or even MUST. I guess the only reason for it not to be MUST is that the lifecycle can technically work around missing targets, but only for linux/amd64 buildpacks.